mods/creative/init.lua

```diff
@@ -80,41 +91,34 @@ local trash = minetest.create_detached_inventory("creative_trash", {
 })
 trash:set_size("main", 1)
 
-creative_inventory.set_creative_formspec = function(player, start_i, pagenum)
+creative_inventory.set_creative_formspec = function(player, start_i, pagenum, tab_idx)
```

Comment:
I like the design.
On Jan 10, 2016 3:33 PM, "kilbith" <notifications@github.com> wrote:

> In mods/creative/init.lua
> <https://github.com/minetest/minetest_game/pull/777#discussion_r49281798>:
>
> > @@ -80,41 +91,34 @@ local trash = minetest.create_detached_inventory("creative_trash", {
> >  })
> >  trash:set_size("main", 1)
> >
> > -creative_inventory.set_creative_formspec = function(player, start_i, pagenum)
> > +creative_inventory.set_creative_formspec = function(player, start_i, pagenum, tab_idx)
>
> Applied.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/minetest/minetest_game/pull/777/files#r49281798>.
>